src/resources/management/how_to_create_and_manage_keys.md

```diff
@@ -0,0 +1,62 @@
+# How to Create and Manage Keys
```

Comment:
oh thats actually a good resource. maybe we should just keep this one, and the other one ^ change it to only full node api key resource